bluesky/callbacks/zmq.py

```diff
@@ -242,12 +260,16 @@ def is_our_message(_hostname, _pid, _RE_id):
     def _poll(self):
         while True:
             message = yield from self._socket.recv()
-            hostname, pid, RE_id, name, doc = message.split(b' ', 4)
+            if self.message_prefix is not None:
+                prefix, hostname, pid, RE_id, name, doc = message.split(b' ', 5)
+            else:
+                hostname, pid, RE_id, name, doc = message.split(b' ', 5)
```

Comment:
I think it should be as in the old code, i.e. `message.split(b' ', 4)`

Reply:
yep thanks!